addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -138,6 +138,12 @@ def setUpClass(cls, chart_template_ref="in"):
         cls.invoice_cash_rounding.action_post()
 
     def test_edi_json(self):
+        # line1: 1000, 10% discount and a tax of 5%
+        # line2: 1000, 10% discount and 4 taxes: 5%, 1.591, 6% SGST 6% CGST, each one affecting the base:
+        # 900 * 1.05 = 945
+        # 945 + 1.591 ~= 946.59
+        # 946.59 * 0.06 = 56.80
+        # total tax: 160.19
```

Comment:
I suppose that, and all the changes of the l10n_in tests files, were validated with some indian colleague?

Reply:
@qdp-odoo The tests in l10n_in are pointless ... The tests are made with 5 taxes instead of 3 with some of them not affected by previous taxes in the middle. The "real" indian case is tested in test_tax

Reply:
@qdp-odoo Confirmed: this setup is fuck-up and is clearly not a valid indian case. The only one is:
x% CGST + x% SGST [+ additional %]